tensorflow/python/ops/nn_ops.py

```diff
@@ -1294,7 +1294,7 @@ def conv3d_transpose(value,
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
       See the @{tf.nn.convolution$comment here}
     data_format: A string, either `'NDHWC'` or `'NCDHW`' specifying the layout
-      of the input and output tensors. Defaults to `'NDHWC'`.
```

Comment:
>  And probably introduce some flexibility with channels_first, channels_last and all these NHWC formats.

We will do that, this is planned.

> Either we should keep this docstring or we should modify the method signatures of all these ops to have a default data_format, which is a string instead of NoneType. 

Having a string as default would be preferable, *unless* we want to make the default globally configurable (change it once and have it apply to all ops for which no data format is specified).